src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Ball;
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(100);
```

Comment:
bound parameter of `nextint()` is exclusive which means that max number we can get right now is 99, we should fix that and make that number a constant